rticonnextdds_connector_test.go

```diff
@@ -143,81 +143,171 @@ func TestDataFlow(t *testing.T) {
 	assert.Nil(t, err)
 
 	// Take any pre-existing samples from cache
-	assert.Nil(t, input.Take())
+	// err = input.Take()
```

Comment:
Why are we commenting this code?

Reply:
We may not need to do it as we create an input and an output right before this. We can remove this.

Reply:
Yes. Let's delete it then 🎉